tests/Fixtures/TestBundle/Entity/DummyWithArrayOfNotResourceObject.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Tests\Fixtures\TestBundle\Dto\NonResourceClass;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ApiResource(filters: ['my_dummy.property'])]
+#[ORM\Entity]
+class DummyWithArrayOfNotResourceObjects
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private $id;
+
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    public function getNotResourceObject(): NonResourceClass
```

Comment:
yes but... with objects from doctrine, it is a little bit more tricky to get an array of objects like this - would you accept if I use a `OneToMany` and I expose the inner array? otherwise we'd need some an  output DTO